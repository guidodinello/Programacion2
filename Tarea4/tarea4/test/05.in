# Implementar operaciones de colaBinarios
insertarEnBinario (25,1.0) 
insertarEnBinario (15,1.0) 
insertarEnBinario (5,1.0) 
insertarEnBinario (45,1.0) 
insertarEnBinario (35,1.0) 
estaVaciaColaBinarios
encolar 5
estaVaciaColaBinarios
frente
encolar 4
estaVaciaColaBinarios
frente
encolar 25
encolar 50
estaVaciaColaBinarios
frente
desencolar
frente
estaVaciaColaBinarios
desencolar
frente
desencolar
frente
estaVaciaColaBinarios
desencolar
estaVaciaColaBinarios
Fin
