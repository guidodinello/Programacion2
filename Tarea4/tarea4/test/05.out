1>#  Implementar operaciones de colaBinarios.
2>Insertado.
3>Insertado.
4>Insertado.
5>Insertado.
6>Insertado.
7>q está vacia.
8>Encolado.
9>q NO está vacia.
10>5
11>Encolado.
12>q NO está vacia.
13>5
14>Encolado.
15>Encolado.
16>q NO está vacia.
17>5
18>Desencolado.
19>Binario vacío.
20>q NO está vacia.
21>Desencolado.
22>25
23>Desencolado.
24>Binario vacío.
25>q NO está vacia.
26>Desencolado.
27>q está vacia.
28>Fin.
